src/crimson/os/seastore/btree/btree_range_pin.cc

```diff
@@ -15,7 +15,7 @@ BtreeNodeMapping<key_t, val_t>::get_logical_extent(
   assert(parent->is_valid());
   assert(pos != std::numeric_limits<uint16_t>::max());
   auto &p = (FixedKVNode<key_t>&)*parent;
-  auto v = p.get_logical_child(ctx, pos);
+  auto v = p.template get_child<LogicalCachedExtent>(ctx, pos, get_key());
```

Comment:
can we assert here parent should be FixedKVLeafNode?

Reply:
will try.